src/plot.jl

```diff
@@ -0,0 +1,33 @@
+@recipe function f(tt::Table; indices = [1,2])
+    if length(indices) == 2
+        x_name, y_name = propertynames(tt)[indices]
+        xguide --> string(x_name)
```

Comment:
What does the `-->` operator do here?

Reply:
it's "use this value if the user didn't provide a different value"